ansible/doc/README.testbed.Internal.md

```diff
@@ -0,0 +1,129 @@
+# Testbed internals
+
+## Topology definition
+
+ - List of currently defined topologies in veos inventory file
+```
+[servers:vars]
+topologies=[‘t1', ‘t1-lag', 't0', 'ptf32', 'ptf64']
```

Comment:
It was changed. Latest list is: 't1', 't1-lag', 't0', 'ptf32', 'ptf64', 't0-64', 't0-64-32'

see sonic-mgmt\ansible\veos